src/data/markdown/translated-guides/en/03 Using k6 browser/02 Running browser tests.md

```diff
@@ -25,55 +25,69 @@ To run a simple local script:
   <CodeGroup labels={["script.js"]} lineNumbers={[true]}>
 
   ```javascript
-  import { chromium } from 'k6/experimental/browser';
+  import { browser } from 'k6/experimental/browser';
+
+  export const options = {
+    scenarios: {
+      ui: {
+        executor: 'shared-iterations',
+        options: {
+          browser: {
+            type: 'chromium',
+          },
+        },
+      },
+    },
+    thresholds: {
+      checks: ["rate==1.0"]
+    }
+  }
 
   export default async function () {
-    const browser = chromium.launch({
-      headless: false,
-      timeout: '60s', // Or whatever time you want to define
-    });
-    const page = browser.newPage();
+    const context = browser.newContext();
+    const page = context.newPage();
 
     try {
       await page.goto('https://test.k6.io/');
       page.screenshot({ path: 'screenshot.png' });
     } finally {
       page.close();
-      browser.close();
     }
   }
   ```
 
   </CodeGroup>
 
-  The preceding code imports the `chromium` [BrowserType](/javascript-api/k6-experimental/browser/browsertype/) (currently the only available `BrowserType` implementation), and uses its `launch` method to start up a Chromium [Browser](/javascript-api/k6-experimental/browser/) process. Two parameters are passed to it. One is the `headless` parameter with the value `false` so you can see the browser launching, and `timeout` parameter with the value `60s` which will be the timeout used for various actions and navigation. For a full list of parameters that you can pass, check out the documentation for [BrowserType.launch()](/javascript-api/k6-experimental/browser/browsertype/launch/).
-  
-  After it starts, you can interact with it using the [browser-level APIs](/javascript-api/k6-experimental/browser/#browser-level-apis). This example visits a test URL and takes a screenshot of the page. Afterwards, it closes the page and the browser.
+  The preceding code imports the `browser` ([BrowserType](/javascript-api/k6-experimental/browser/browsertype/)), and uses its `newContext` method to starts a new Chromium [BrowserContext](/javascript-api/k6-experimental/browser-context/), which is an incognito window. 
```

Comment:
Should this reference to `BrowserType` link to the "browser module" instead?

Also, I would not mention that a new browser process is launched on `newContext` as that seems to refer to a technical detail that is not exactly true and will change soon once we integrate with the k6 events, right?

Reply:
Agreed :) See 0228368f532fa89c688b1c5658e1a03f7c9c0661